client/index.html

```diff
@@ -25,7 +25,7 @@
 	<meta name="theme-color" content="#455164">
 
 	</head>
-	<body class="signed-out <%= public ? "public" : "" %>">
+	<body class="signed-out {{ternary public 'public' '' }}">
```

Comment:
Yeah, I saw them as well, which was good, but it didn't work. I also can't find it anywhere on their official documentation. I imagine it was something that worked a while ago but was never deliberate. If I try it, I get an error now.